npm warn ERESOLVE overriding peer dependency

After upgrading to React 19 and then upgrading `@adamscybot/react-leaflet-component-marker` and `react-leaflet`:
```
 @adamscybot/react-leaflet-component-marker   ^2.0.1  →   ^2.0.2
 @types/leaflet                              ^1.9.12  →  ^1.9.16
 react-leaflet                                ^4.2.1  →   ^5.0.0
```

I'm getting the following warning. 
```
npm warn ERESOLVE overriding peer dependency

npm warn While resolving: react-reverse-portal@2.1.2
npm warn Found: react@19.0.0
npm warn node_modules/react
npm warn   react@"19.0.0" from the root project
npm warn   46 more (@floating-ui/react-dom, @mdx-js/react, ...)
npm warn
npm warn Could not resolve dependency:
npm warn peer react@"^16.0.0 || ^17.0.0 || ^18.0.0" from react-reverse-portal@2.1.2
npm warn node_modules/@adamscybot/react-leaflet-component-marker/node_modules/react-reverse-portal
npm warn   react-reverse-portal@"^2.1.2" from @adamscybot/react-leaflet-component-marker@2.0.2
npm warn   node_modules/@adamscybot/react-leaflet-component-marker
npm warn
npm warn Conflicting peer dependency: react@18.3.1
npm warn node_modules/react
npm warn   peer react@"^16.0.0 || ^17.0.0 || ^18.0.0" from react-reverse-portal@2.1.2
npm warn   node_modules/@adamscybot/react-leaflet-component-marker/node_modules/react-reverse-portal
npm warn     react-reverse-portal@"^2.1.2" from @adamscybot/react-leaflet-component-marker@2.0.2
npm warn     node_modules/@adamscybot/react-leaflet-component-marker

npm warn ERESOLVE overriding peer dependency

npm warn While resolving: react-reverse-portal@2.1.2
npm warn Found: react-dom@19.0.0
npm warn node_modules/react-dom
npm warn   react-dom@"19.0.0" from the root project
npm warn   19 more (@floating-ui/react-dom, @radix-ui/react-arrow, ...)
npm warn
npm warn Could not resolve dependency:
npm warn peer react-dom@"^16.0.0 || ^17.0.0 || ^18.0.0" from react-reverse-portal@2.1.2
npm warn node_modules/@adamscybot/react-leaflet-component-marker/node_modules/react-reverse-portal
npm warn   react-reverse-portal@"^2.1.2" from @adamscybot/react-leaflet-component-marker@2.0.2
npm warn   node_modules/@adamscybot/react-leaflet-component-marker
npm warn
npm warn Conflicting peer dependency: react-dom@18.3.1
npm warn node_modules/react-dom
npm warn   peer react-dom@"^16.0.0 || ^17.0.0 || ^18.0.0" from react-reverse-portal@2.1.2
npm warn   node_modules/@adamscybot/react-leaflet-component-marker/node_modules/react-reverse-portal
npm warn     react-reverse-portal@"^2.1.2" from @adamscybot/react-leaflet-component-marker@2.0.2
npm warn     node_modules/@adamscybot/react-leaflet-component-marker

```

react-leaflet-component-marker depends on `react-reverse-portal@2.1.2`
See: https://github.com/adamscybot/react-leaflet-component-marker/blob/main/package.json#L62

`react-reverse-portal@2.1.2` doesn't seem to support React v19 yet:
https://github.com/httptoolkit/react-reverse-portal/blob/master/package.json#L56-L57

Looks like the author is open for PR's:
https://github.com/httptoolkit/react-reverse-portal/issues/46

--- a/package.json
+++ b/package.json
@@ -59,7 +59,7 @@
   },
   "dependencies": {
     "react-is": "^18.3.1",
-    "react-reverse-portal": "^2.1.2",
+    "react-reverse-portal": "^2.2.0",
     "type-fest": "^4.30.2"
   },
   "release": {

